packages/dashboard-frontend/src/pages/GetStarted/CustomWorkspaceTab/index.tsx

```diff
@@ -49,44 +52,51 @@ export class CustomWorkspaceTab extends React.PureComponent<Props, State> {
     super(props);
 
     const devfile = this.buildInitialDevfile();
-    const storageType = attributesToType(devfile.attributes);
-    const workspaceName = devfile.metadata.name ? devfile.metadata.name : '';
-    const generateName = !workspaceName ? devfile.metadata.generateName : '';
+    const storageType = isCheDevfile(devfile) ? attributesToType(devfile.attributes) : 'persistent';
+    const workspaceName = devfile.metadata.name ? devfile.metadata.name || '' : '';
+    const generateName = isCheDevfile(devfile) && !workspaceName ? devfile.metadata.generateName : '';
     this.state = { devfile, storageType, generateName, workspaceName, isCreated: false };
     this.devfileEditorRef = React.createRef<Editor>();
   }
 
-  private buildInitialDevfile(generateName?: string): che.WorkspaceDevfile {
-    const devfile = {
+  private buildInitialDevfile(generateName ='wksp-'): Devfile {
+    const devfile = (this.props.workspacesSettings['che.devworkspaces.enabled'] === 'true') ? {
+        schemaVersion: '2.1.0',
```

Comment:
It does not work for me:
![Screenshot_20210914_161943](https://user-images.githubusercontent.com/5887312/133266706-13a82437-79f5-41dd-9fc2-a36227e549bd.png)

it's more problem on JSON Schema side, but to workaround it we can proceed with:
```
schemaVersion: 2.1.0
metadata:
  name: wksp-zpet
components: []
```
^ it works.

Reply:
I have fixed it. Now it works without adding 'components: []'